README.md

```diff
@@ -1,2 +1,8 @@
 # wavefront-sdk-javascript
 Wavefront Core Javascript SDK
+
+### Usage
+Run `npm run build` to generate a UMD bundle under `dist` directory.
+
+
+To run with debugger, put `-r @babel/preset-env -r @babel/register -r regenerator-runtime` under the node parameters.
```

Comment:
would be nice to put this under the "start" script in package.json, so you can `npm start` this.